### ISSUE TITLE
request for official m3u list syntax

### ISSUE THREAD

**Issue by Brenders** [linuxmint/hypnotix#200]
Hi, I wanted to know if you could have a link with some official guidelines on the syntax of m3u files, I would like to be able to better understand how to create lists for hypnotix with the right syntax (I am an active contributor to free iptv, and have also contributed to hypnotix a few times) as I have not found many nowhere and I don't know where to find this information regarding the syntax you use in hypnotix.
have a good day.

**Comment by Axel-Erfurt** [comment]
It always depends on what you want.

Playlist must start with 

```
#EXTM3U
```

For a channel you always need 2 lines, 
line 1 for name and extra informations
line 2 for the url

If you need groups and / or logos, you have to indicate them.

for example

```
#EXTINF:-1 tvg-name="Das Erste HD" tvg-id="ARD.de" tvg-logo="http://logos.zattic.com/images/channels/logos/2a2a4aee64bbd6f7f817/black/84x48.png" group-title="GERMAN TV",Das Erste HD
https://mcdn.daserste.de/daserste/de/master.m3u8
```

If not, you can keep it simple.  Only name and url.

```
#EXTINF:-1,Das Erste HD
https://mcdn.daserste.de/daserste/de/master.m3u8
```

**Comment by Brenders** [comment]
thanks Axel-Erfurt for the exhaustive answer, the tvg-id must contain the main site of the channel? what syntax should i use about movies and series?

**Comment by Axel-Erfurt** [comment]
For movies use group-title="VOD"

```
#EXTM3U
#EXTINF:-1 tvg-name="Movie Name" group-title="VOD", Movie Name
```

for series:


use 
group-title="SERIES Series Name"
tvg-name="SxxExx Episode Title"

S = Series
E = Episode

S04E06 = Series 4 Episode 6

```
#EXTM3U
#EXTINF:-1 tvg-name="Frasier S04E06 Episode Title" group-title="SERIES Frasier",
```

**Comment by Brenders** [comment]
I wanted to fix [zz_movies](https://github.com/Free-TV/IPTV/blob/master/zz_movies.mdl) because it doesn't appear in the movies section, group-title = "VOD" can also contain (EN) transmitting it in group-title="VOD (EN)"?

Thanks for everything, I promise this will be the last question and sorry for the inconvenience

**Comment by lulol** [comment]
Wouldn't it be interesting to add all this info to the README.md so that is readily available? #47 #170

**Comment by dasaferi** [comment]
> Wouldn't it be interesting to add all this info to the README.md so that is readily available? #47 #170

issues #170 are already fixed with a push.

**Comment by random-pixels** [comment]
Do you know what the 
#EXTINF:-1
means? I have seen different values there like 0 and 1. Just wondering what the number means.
